docs/nodes/validate/add-a-validator.md

```diff
@@ -332,16 +257,92 @@ $ ts-node examples/platformvm/getPendingValidators.ts
 The response should include the node we just added:
 
 ```json
-{ validators: [{
-"nodeID": "NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg",
-"startTime": "1654656829",
-"endtime": "1662602029",
-"stakeAmount": "1000000000"
-}], delegators: [] }
+{
+  "validators": [
+    {
+      "nodeID": "NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg",
+      "startTime": "1654656829",
+      "endtime": "1662602029",
+      "stakeAmount": "1000000000"
+    }
+  ],
+  "delegators": []
+}
 ```
 
 When the time reaches `1654656829` (Wed Jun 08 2022 02:53:49 GMT+0000), this node will start validating the Primary Network. When it reaches `1662602029` (Thu Sep 08 2022 01:53:49 GMT+0000), this node will stop validating the Primary Network. The staked AVAX will be returned to the user's `P-chain address`, and the rewards, if any, will be given to `rewardAddress`.
 
+#### Customizing Addresses
+
+There are 4 addresses which are needed when calling `pchain.buildAddValidatorTx`. Only 2 of them can be changed: `toAddresses` and `rewardAddresses`. For backward-compatibility reasons, `fromAddresses` and `changeAddresses`
+are just placeholders and are ignored.
+
+`toAddresses`
+
+An array of addresses who receive the staked tokens at the end of the staking period.
+
+`rewardAddresses`
+
+When a validator stops validating the Primary Network, they will receive a reward if they are sufficiently responsive and correct while they validated the Primary Network. These tokens are sent to `rewardAddresses`. The original stake will be sent back to the addresses defined in `toAddresses`.
+
+A validator’s stake is never slashed, regardless of their behavior; they will always receive their stake back when they’re done validating.
+
+If you want to customize the `to`, `from`, `change`, and `reward` addresses, please adapt the example below into your project.
+
+Locate this part of the code
+
+```ts
+let privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`;
```

Comment:
In our example code we don't use semicolons so please remove the semicolons from each of these example snippets.

Reply:
@Julian-dev28 please update